test/config.ts

```diff
@@ -1,11 +1,15 @@
 import ts from "typescript";
 import tsThree from "typescript-3";
 import tsFourSeven from "typescript-4.7";
+import tsFiveFive from "typescript-5.5";
+import tsFiveSix from "typescript-5.6";
 import path from "node:path";
 
 export const tsModules = <const>[
   ["3.6.5", tsThree, "typescript-3"],
   ["4.7.4", tsFourSeven, "typescript-4.7"],
+  ["5.5.4", tsFiveFive, "typescript-5.5"],
+  ["5.6.3", tsFiveSix, "typescript-5.6"],
   ["Latest", ts, "typescript"],
```

Comment:
Technically, 5.6.3 and `latest` are the same thing right now. However, I wanted to explicitly define 5.6.3 so we run tests against it. Then, when 5.7++ come out, we'll get test coverage.